packages/builder/src/components/design/settings/controls/ButtonActionEditor/actions/NavigateTo.svelte

```diff
@@ -7,24 +7,28 @@
 </script>
 
 <div class="root">
-  <Label small>Screen</Label>
+  <Label small>Screen or URL</Label>
```

Comment:
The changes here look good. 

The experience here I think might be confusing for users though for these reasons:
1. The form options suggest you can click both open in new tab and in modal, but that seems incompatible
2. The open in new tab mentions external link only - opening a screen in a new tab could be a valid case

I think ideally you have separate actions for: 
- `Navigate To Screen`
   - Screen input as is today 
   - Open in: (radio selection)
      - Modal 
      - New tab

- `Navigate To External URL`
   - URL input, with validation for protocol
   - Open in new tab (checkbox) 

Alternatively you could combine them into the existing action as is the case at the minute, but provide a picker at the top for destination:
- Screen
- External URL

Reply:
Unrelated: I think the navigate to screen is a point of confusion for first time users, particularly around needing to build up the binding yourself e.g. `/fleet/{{ New Repeater.Fleet._id }}`. 

By splitting screen from url we give ourselves a bit more opportunity to improve the screen select e.g. by listing all the screens in a dropdown and rendering inputs for the individual path variables or adding params

Reply:
I updated the action to include a dropdown for the destination type. Updated the screenshots in the OP.

Reply:
![Screenshot 2023-02-28 at 13 11 20](https://user-images.githubusercontent.com/101575380/221864054-d36e62bb-2b02-4256-9ec1-a0c30ee83cde.png)

When there are path variables, it just adds the text directly. The user would need to update these. It possibly could be nice to parse the path variables out and allow the user to enter bindings if you think it's worth doing that, although getting what's already done merged in would still be an improvement over what's there now.

Also a small UI question. Do you think I should remove the label from the binding box. The *Destination* dropdown above already tells the user which type of input it is.

Reply:
All good, thanks for clarifying. I think what you have there is great - a big improvement on what we have now. Updating the binding yourself should be more clear now. 

I think the label above the box is fine, but removing it might also be fine lol. What does it look like without it?

Reply:
![Screenshot 2023-02-28 at 13 52 42](https://user-images.githubusercontent.com/101575380/221874061-17f4fbf8-2688-4383-ba13-8a634d153a8c.png)

![Screenshot 2023-02-28 at 13 53 03](https://user-images.githubusercontent.com/101575380/221874151-ecaafaf0-c598-4de4-8439-9e67e689939c.png)

Looks a bit neater imo. What do you think?

Reply:
no label LGTM